artview/components/select_region.py

```diff
@@ -247,44 +247,17 @@ def viewTable(self):
 
     def saveTable(self):
         '''Save a Table of SelectRegion points to a CSV file.'''
-        data = self.VSelectRegion.value
-        fsuggest = 'SelectRegion_' + self.getField() + '_' + \
-            str(data[:, 0].mean()) + '_' + \
-            str(data[:, 1].mean())+'.csv'
-        path = QtGui.QFileDialog.getSaveFileName(
-                self, 'Save CSV Table File', fsuggest, 'CSV(*.csv)')
-        if not path.isEmpty():
-            with open(unicode(path), 'wb') as stream:
-                writer = csv.writer(stream)
-                writer.writerow(self.columns)
-                for row in range(data.shape[0]):
-                    rowdata = []
-                    for column in range(data.shape[1]):
-                        item = data[row][column]
-                        if item is not None:
-                            rowdata.append(
-                                unicode(item).encode('utf8'))
-                        else:
-                            rowdata.append('')
-                    writer.writerow(rowdata)
-        # Commented out below is alternative ascii output (needs reformating)
-# +                outfile = open(fsuggest,'w')
-# +                outfile.write(
-# "     X        Y    Azimuth   Range     Value   Az Index  R Index\n")
-# +                    outfile.write(
-# "%8.2f %8.2f %8.2f %8.3f %8.2f %8d %8d\n" %(self.xys[self.ind[i],0],
-# self.xys[self.ind[i],1], self.az[self.ind[i]/self.r.size],
-# self.r[self.ind[i]%self.r.size],
-# self.Vradar.fields[self.Vfield.value][
-# 'data'][self.Vradar.sweep_start_ray_index['data'][
-# self.Vtilt]+self.ind[i]/self.r.size,self.ind[i] \
-# % self.r.size],self.Vradar.sweep_start_ray_index['data'][self.Vtilt]+\
-# self.ind[i]/self.r.size,self.ind[i]%self.r.size))
-# +                outfile.close()
-#  msg = "     X        Y    Azimuth   Range     Value   Az Index  R Index\n"
-#                for i in range(self.ind.size):
-#                warn = common.ShowWarning(msg)
-#                    print "%8.2f %8.2f %8.2f %8.3f %8.2f %8d %8d" %\
+        points = self.Vpoints.value
+        if points is not None:
+            fsuggest = 'SelectRegion_' + self.getField() + '_' + \
+                str(points.axes['x_disp']['data'][:].mean()) + '_' + \
+                str(points.axes['y_disp']['data'][:].mean())+'.csv'
+            path = QtGui.QFileDialog.getSaveFileName(
+                    self, 'Save CSV Table File', fsuggest, 'CSV(*.csv)')
```

Comment:
Does fsuggest work for you? It does nothing on my system, which is why I abandoned it a while back.